examples/e-commerce/src/app.css

```diff
@@ -125,6 +125,10 @@ mark {
   margin-right: 5px;
 }
 
+.ais-ClearRefinements-button:disabled {
```

Comment:
This should have been caught with:

https://github.com/algolia/angular-instantsearch/blob/764960b67a05fa0e8609c47fa45984fac84d2ef5/examples/e-commerce/src/theme.css#L13-L15

but apparently wasn't because Angular components are not targeted with this selector. Can we find a selector in the theme that works for Angular as well?

Reply:
```css
[class^='ais-'][class$='--disabled'],
[class^='ais-'][class$='--disabled'] * {
  cursor: not-allowed;
}
```

This works. What do you think?

```html
<button class="ais-ClearRefinements-button ais-ClearRefinements-button--disabled" disabled=""> Clear filters </button>
```

Reply:
Fine to me! We'll make sure to replicate this change in the other examples.

Reply:
Okay 👍

Reply:
529b21b